src/pynwb/core.py

```diff
@@ -93,8 +93,8 @@ class ScratchData(NWBData):
     __nwbfields__ = ('notes',)
 
     @docval({'name': 'name', 'type': str, 'doc': 'the name of this container'},
-            {'name': 'data', 'type': ('array_data', 'data', Data), 'doc': 'the source of the data'},
-            {'name': 'notes', 'type': str, 'doc': 'notes about the data', 'default': ''})
+            {'name': 'data', 'type': ('scalar_data', 'array_data', 'data', Data), 'doc': 'the source of the data'},
+            {'name': 'notes', 'type': str, 'doc': 'notes about the data'})
```

Comment:
While we are changing things for Scratch, I think it would make sense to change ``notes`` for ``ScratchData`` to ``description`` for consistency with ``add_scratch``

Reply:
Done